### PR TITLE
Fixes a slime runtime

### PR DESCRIPTION
**Pull request by kriskog**
## About The Pull Request
```
Runtime in life.dm,19: Cannot read null.reagent_list
  proc name: Life (/mob/living/simple_animal/slime/Life)
```

## Why It's Good For The Game
Bug fix

## Changelog
:cl: Skoglol
fix: Fixed a slime runtime.
/:cl:

### CODE REVIEW

**Comment by nemvar** [comment]
Thanks

File: code/modules/mob/living/simple_animal/slime/life.dm [hunk 1]
```diff
@@ -16,6 +16,8 @@
 			handle_feeding()
 		if(!stat) // Slimes in stasis don't lose nutrition, don't change mood and don't respond to speech
 			handle_nutrition()
+			if(QDELETED(src)) // Stop if the slime split during handle_nutrition()
```
Comment (ninjanomnom) [inline code/modules/mob/living/simple_animal/slime/life.dm]: Why is it not stopping the life ticks when it gets qdeleted, this is a bad way to fix the issue

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/slime/life.dm (hunk 1 above)
Comment (kriskog) [reply code/modules/mob/living/simple_animal/slime/life.dm]: It is stopping the ticks, this happens once after split. So it's not stopping the tick currently in progress.

Comment (ninjanomnom) [reply code/modules/mob/living/simple_animal/slime/life.dm]: Oh is this happening in handle_nutrition?

**Review by ninjanomnom — APPROVED** [review]

File: code/modules/mob/living/simple_animal/slime/life.dm (hunk 1 above)
Comment (kriskog) [reply code/modules/mob/living/simple_animal/slime/life.dm]: Yeah, that's it. I guess the comment could have been more clear about current tick.

**Pull request merged**